### ISSUE TITLE
Features Suggestion. Convert text objects to paths.

### ISSUE THREAD

**Issue by SciutoAlex** [paperjs/paper.js#1323]
Hi there,

Thanks for all the hard work you put into paper.js. I really like using it to make procedurally generated projects. I'm working on a project now where I'm trying to boolean combine text shapes with a compound path. I can't see a way to convert PointText to something that performs boolean operations. I'm planning on loading individual characters via SVG to perform the combinations, but that is going to be a kerning pain!

I can see this feature request being pretty difficult, but I thought I'd suggest it.

![image](https://cloud.githubusercontent.com/assets/1485679/25971044/e3420034-3668-11e7-95d0-4307d48a3734.png)

Alex

**Comment by lehni** [comment]
Yeah this is on the road map since a long while... 

This looks like a good candidate as a library to rely on to do all the heavy font lifting: https://github.com/devongovett/fontkit

**Comment by SciutoAlex** [comment]
<img width="369" alt="screen shot 2017-05-12 at 1 09 50 pm" src="https://cloud.githubusercontent.com/assets/1485679/26008865/86545a32-3714-11e7-8669-a071db334b64.png">

Thanks for the reply. I ended up using [Text-to-SVG](https://www.npmjs.com/package/text-to-svg). It worked out well, but I had to create a little server script because it didn't run well in browser. Probably best in the future to rely on the fontkit library.
